community/kernel/src/test/java/org/neo4j/kernel/impl/store/NeoStoresTest.java

```diff
@@ -911,15 +947,15 @@ else if ( rel == rel2 )
         return count;
     }
 
-    private PropertyReceiver<DefinedProperty> newPropertyReceiver( final ArrayMap<Integer,Pair<DefinedProperty,Long>> props )
+    private PropertyReceiver<DefinedProperty> newPropertyReceiver(
+            final ArrayMap<Integer,Pair<DefinedProperty,Long>> props )
     {
-        return ( property, propertyRecordId ) -> props.put( property.propertyKeyId(), Pair.of( property, propertyRecordId ) );
+        return ( property, propertyRecordId ) -> props
+                .put( property.propertyKeyId(), Pair.of( property, propertyRecordId ) );
```

Comment:
Here I'd prefer to either break before `props`, or at the start of the parameter list.